src/realm/sync/subscriptions.cpp

```diff
@@ -400,12 +407,13 @@ SubscriptionStore::SubscriptionStore(DBRef db)
 
         auto schema_metadata = tr->add_table(c_flx_metadata_table);
         auto version_col = schema_metadata->add_column(type_Int, c_flx_meta_schema_version_field);
-        schema_metadata->create_object().set(version_col, int64_t(1));
+        schema_metadata->create_object().set(version_col, int64_t(2));
 
         auto sub_sets_table =
             tr->add_table_with_primary_key(c_flx_subscription_sets_table, type_Int, c_flx_sub_sets_version_field);
         auto subs_table = tr->add_embedded_table(c_flx_subscriptions_table);
         m_sub_keys->table = subs_table->get_key();
+        m_sub_keys->id = subs_table->add_column(type_ObjectId, c_flx_sub_id_field);
```

Comment:
Do we want to make the id a primary key or not?

Reply:
If you changed this to be a primary key, it couldn't be an embedded table so is that back on the table?

Reply:
Oh, didn't realize it's an embedded table, disregard this then 😅